docs/en-US/component/slider.md

```diff
@@ -97,7 +106,8 @@ slider/show-marks
 | step                  | step size                                                                                                 | number          | —                      | 1       |
 | show-input            | whether to display an input box, works when `range` is false                                              | boolean         | —                      | false   |
 | show-input-controls   | whether to display control buttons when `show-input` is true                                              | boolean         | —                      | true    |
-| input-size            | size of the input box                                                                                     | string          | large / default /small | default |
+| size                  | size of the slider                                                                                        | string          | large / default /small | default |
```

Comment:
I think you have missed a space after `default /`,

Reply:
Sure